tests/pipeline/snapshots/snap_test_pipeline.py

```diff
@@ -563,7 +563,7 @@
             "prefix": "resources-pipeline-with-paths-",
             "repoConfig": {
                 "repoAuthFlags": {
-                    "caFile": "/my-cert.cert",
+                    "caFile": str(Path("/my-cert.cert")),
```

Comment:
did you manually edit the snapshot? it should only be generated

Reply:
I updated it manually, we don't have any other way to do that for the snapshots. At least one I am aware of. Any ideas ?

Reply:
how about changing the path to a relative one in tests/pipeline/resources/pipeline-with-paths/pipeline.yaml, would that work?